.github/workflows/ci.yml

```diff
@@ -78,7 +78,8 @@ jobs:
           toolchain: ${{ env.RUST_VERSION }}
       - uses: Swatinem/rust-cache@v2
       - uses: arduino/setup-protoc@v2
-      - run: cargo run --bin sozo -- --manifest-path crates/dojo-core/Scarb.toml test
+      - run: curl -L https://install.dojoengine.org | bash && source /home/runner/.bashrc && echo $PATH && /home/runner/.config/.dojo/bin/dojoup
```

Comment:
perhaps it makes sense to use the dojo docker image for this job? it comes with katana installed

Reply:
That definitely makes sense.

By the way I noticed that dojoup is downloaded to `/home/runner/.config/.dojo/bin/dojoup`, not sure if it's the expected behaviour